.vscodeignore

```diff
@@ -1,11 +1,12 @@
-.vscode/**
-.vscode-test/**
-src/**
-dist/**
-node_modules/**
-.gitignore
-.yarnrc
-**/tsconfig.json
-**/.eslintrc.json
-**/*.map
-**/*.ts
+*
+*/**
+# Whitelist the following files
+!package.json
+!language-configuration.json
+!LICENSE
+!README.md
+!CHANGELOG.md
+!images/icon.png
+!out/*.js
+!out/*.wasm
```

Comment:
This seems overly cautious. The entire out folder is expected to go into the plugin.

Reply:
Agree. Main reason was to continue excluding ‘.map’ files (previous blacklist was ‘**/*.map’).

Perhaps a better strategy would be to whitelist ‘out’ and include an ‘exceptions’ section at the bottom of the ignore file to exclude ‘.map’ files. What do you think?

The [discussion on excluding map files in the VSCode Python extension](https://github.com/microsoft/vscode-python/issues/13961) provides more context on reasons for this exclusion. Slowed the extension (at least previously), increased bundle size and only used for debugging to determine exact file and line number that threw an extension - which is not a present concern with this extension given it would apply to two files.